mobly/controllers/android_device_lib/callback_handler.py

```diff
@@ -14,22 +14,18 @@
 
 import time
 
```

Comment:
should we also mark this and the other v1 classes "deprecated" in docstring?

Reply:
Should we mark them as deprecated after setting v2 as default?